client/src/components/LogFileList.js

```diff
@@ -1,7 +1,6 @@
+import WidgetListGroupItem from 'components/WidgetListGroupItem';
 import React from 'react';
 import { Button } from 'react-bootstrap';
-import WidgetListGroupItem from 'components/WidgetListGroupItem';
-import LogFile from '../api/v2/files';
```

Comment:
I believe `LogFile` was imported for its use in the JSDoc, probably best to keep it for now

Reply:
I don't think it really works with JSDoc. If we want it then we would have to define a JSDoc type in `api/v2/files` and import it using JSDoc imports.
![image](https://user-images.githubusercontent.com/50545432/96723425-45e02000-13fa-11eb-8c08-1cba0b3aa3fe.png)